libs/vkd3d/vulkan_procs.h

```diff
@@ -225,6 +225,10 @@ VK_DEVICE_EXT_PFN(vkCmdEndRenderPass2KHR)
 VK_DEVICE_EXT_PFN(vkCmdNextSubpass2KHR)
 VK_DEVICE_EXT_PFN(vkCreateRenderPass2KHR)
 
+/* VK_KHR_bind_memory2 */
+VK_DEVICE_EXT_PFN(vkBindBufferMemory2KHR);
```

Comment:
In the following commit, the legacy entry points should be removed from vulkan_procs.h to ensure that we don't accidentally use the legacy entry points, and verifies that we are not still using them.

Reply:
I can do that but it seems like some legacy entrypoints stil remain like vkCreateRenderPass().